### PR TITLE
use `rmdir` in place of `rm -r` when cleaning up mount points

### PR DESCRIPTION
**Pull request by raingloom**
fixes issue #21

### CODE REVIEW

File: initcpio/hooks/oroot [hunk 1]
```diff
@@ -39,7 +38,7 @@ run_hook() {
               mount -o \$(awk '{if (\$2 == \"/\") {print \$4}}' /etc/fstab) \"$root\" /run/oroot || mount \"$root\" /run/oroot;
               rsync -ax --no-whole-file --inplace --delete / /run/oroot --exclude boot --exclude dev --exclude mnt --exclude proc --exclude run --exclude sys --exclude tmp --exclude usr/bin/overlay_flush;
               umount /run/oroot &&
-              rm -r /run/oroot &&
+              rmdir /run/oroot &&
```
Comment (ugjka) [inline initcpio/hooks/oroot]: Remove the "&&" in this line, otherwise the command fails

Comment (ugjka) [inline initcpio/hooks/oroot]: Remove the "&&" in this line, it causes the command to fail becouse it doesn't connect to anything

Comment (raingloom) [reply initcpio/hooks/oroot]: Isn't it supposed to connect to the next line with the `cryptsetup close`?

Comment (raingloom) [reply initcpio/hooks/oroot]: also, since we are cleaning up, wouldn't it make more sense to use `set -x` instead of concatenating everything with `&&`?

Comment (ugjka) [reply initcpio/hooks/oroot]: Stupid github put the comment on the wrong line, there's one case in the file where there is no cryptsetup close

Comment (ugjka) [reply initcpio/hooks/oroot]: I see you already fixed it https://github.com/bluerider/liveroot/pull/23/commits/7814d4d4269573ad317b7dfc17e1c48348f637e0

Comment (raingloom) [reply initcpio/hooks/oroot]: No prob, and yeah, at least that works now.

**Review by raingloom — COMMENTED** [review]
Don't pull this commit, it got mixed in somehow. Sorry, I'm quite new to working on other people's code.

**Comment by raingloom** [comment]
@bluerider any specific reason this was closed? You still **should** use rmdir when you know the directory will be empty. Wouldn't want to delete anything accidentally put in there, would we?

**Pull request closed without merging**